test/src/e2e/multisig.test.ts

```diff
@@ -39,14 +42,47 @@ import {
     daveSecret,
     faucetAddress
 } from "../helper/constants";
+import { AssetTransaction } from "codechain-sdk/lib/core/Transaction";
 import CodeChain from "../helper/spawn";
+import { ChainRpc } from "codechain-sdk/lib/rpc/chain";
+import { Key } from "codechain-sdk/lib/key";
 
 const { PUSH, PUSHB, CHKMULTISIG } = Script.Opcode;
 
+// If one only sends certainly failing trasactions, the miner would not generate any block.
+// So to clearly check the result failed, insert the failing transactions inbetween succeessful ones.
+async function sendFailingTx(
+    node: CodeChain,
+    targetTx: Transaction & AssetTransaction
+): Promise<{
+    signedDummyTx: SignedTransaction;
+    targetTxHash: H256;
+}> {
+    await node.sdk.rpc.devel.stopSealing();
+
+    const blockNumber = await node.getBestBlockNumber();
+    const seq = await node.sdk.rpc.chain.getSeq(faucetAddress);
+    const pay = await node.sendPayTx({ seq, quantity: 1 });
+    const targetTxHash = await node.sendAssetTransaction(targetTx, {
+        seq: seq + 1
+    });
+
+    await node.sdk.rpc.devel.startSealing();
+    await node.waitBlockNumber(blockNumber + 1);
+
+    return {
```

Comment:
How about checking whether the dummy transaction included in this function instead of returning it?

Reply:
Ok. That seems better.